.github/workflows/check_code_format.yml

```diff
@@ -0,0 +1,25 @@
+#  This workflow will run the npm run check script to check if the formatting of the code is correct or not.
+name : check-code-formatting
+on:
+  push:
+    branches:
+      - '**'
+
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
where's this `matrix`? Are you sure this workflow will work?
Did you try in your fork?

Please check this one: https://github.com/Real-Dev-Squad/website-backend/blob/develop/.github/workflows/test.yml

We don't need it to run on 3/4 node versions, just one would be fine (14.x) since this is just checking the formatting and not testing anything as such

Reply:
My bad, I thought it was something available since it was like that in the ref shared in the issue. Fixing the same

Reply:
Fixed the same. Thanks for the reference